com.unity.render-pipelines.core/Documentation~/Common/lens-flare-data-driven-asset.md

```diff
@@ -145,7 +148,8 @@ The Lens Flare Element asset has the following properties:
 | ---------------- | ------------------------------------------------------------ |
 | Colors           | The range of colors that this asset applies to the lens flares. You can use the **Position Spacing** curve to determine how this range affects each lens flare. |
 | Position Spacing | Adjust this curve to change the placement of the lens flare elements in the **Lens Spread**. |
-| Scale Variation  | Adjust this curve to control the size range of the lens flare elements. |
+| Rotation | Uniform angle (in degree) used with multiple element enabled with Curve distribution. |
```

Comment:
| Rotation | The uniform angle of rotation (in degrees) applied to each element distributed along the curve. |

Reply:
Please update the screenshot to be accurate to these property changes.